README.md

```diff
@@ -1,9 +1,20 @@
-[![Build Status](https://travis-ci.org/oasis-roles/wipefs.svg?branch=master)](https://travis-ci.org/oasis-roles/wipefs)
+[![Build Status](https://travis-ci.org/oasis-roles/disk_wipe.svg?branch=master)](https://travis-ci.org/oasis-roles/disk_wipe)
 
-wipefs
+disk_wipe
 ===========
 
-Basic description for wipefs
+This role is not idempotent and is intended to be used when you are preparing nodes
```

Comment:
During yesterday's discussion we agreed that we should have the README recommend to use this role in a separate, isolated playbook to prevent disks from getting wiped a second time.

Reply:
updated